sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableAsyncClient.java

```diff
@@ -993,21 +1113,28 @@ private AccessPolicy toAccessPolicy(TableAccessPolicy tableAccessPolicy) {
     }
 
     /**
-     * Executes all operations within the list inside a transaction. When the call completes, either all operations in
-     * the transaction will succeed, or if a failure occurs, all operations in the transaction will be rolled back.
-     * Each operation must operate on a distinct row key. Attempting to pass multiple operations that share the same
-     * row key will cause an error.
-     *
-     * @param transactionActions A list of {@link TableTransactionAction transaction actions} to perform on entities
-     * in a table.
-     *
-     * @return A reactive result containing a list of {@link TableTransactionActionResponse sub-responses} that
-     * correspond to each operation in the transaction
-     *
-     * @throws IllegalStateException If no operations have been added to the list.
-     * @throws TableTransactionFailedException if any operation within the transaction fails. See the documentation
-     * for the client methods in {@link TableClient} to understand the conditions that may cause a given operation to
-     * fail.
+     * Executes all {@link TableTransactionAction actions} within the list inside a transaction. When the call
+     * completes, either all {@link TableTransactionAction actions} in the transaction will succeed, or if a failure
+     * occurs, all {@link TableTransactionAction actions} in the transaction will be rolled back. Each
+     * {@link TableTransactionAction action} must operate on a distinct row key. Attempting to pass multiple
+     * {@link TableTransactionAction actions} that share the same row key will cause an error.
+     *
+     * <p><strong>Code Samples</strong></p>
+     * <p>Submits a transaction that contains multiple {@link TableTransactionAction actions} to be applied to
+     * {@link TableEntity entities} on a table. Prints out details of each {@link TableTransactionAction action}'s
+     * {@link Response HTTP response}.</p>
+     * {@codesnippet com.azure.data.tables.tableAsyncClient.submitTransaction#List}
```

Comment:
Having a codesnippet to show how to handle a failed transaction will be useful here.